css/workspace-style.css

```diff
@@ -99,6 +99,10 @@ tbody > .workspace-tr:hover {
 	flex-basis: 0;
 }
 
+.app-workspace .app-navigation-entry__children {
```

Comment:
Could you move it to the style block of the MenuItemSelector component, please?
This will help maintain better organization.

Otherwise, it works ! :+1: